CONTRIBUTING.md

```diff
@@ -16,10 +16,8 @@ The OpenHTF codebase is set up to use pip/virtualenv to manage dependencies.
 Pip is the python package manager, while virtualenv is a tool to isolate python
```

Comment:
This file had some draft changes, but PR #313 contains the latest and greatest text, and will override the stuff in this PR on rebase.